recipe/meta.yaml

```diff
@@ -22,39 +21,46 @@ requirements:
     - pip
     - python
     - setuptools
-    - cython
-    - mpi4py
-    - numpy
+    - cython >=0.22
+    - mpi4py >=2.0.0
+    - numpy >=1.20.0
```

Comment:
I'm not sure about the numpy pin. 

Typically we pin a certain version of numpy to each version of python (see https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy). 

As this package has a run_compatible, and now needs at least numpy 1.20, but most other packages are built against numpy 1.19, this leads to incompatibilities.

Just want to point that out, not sure whether anything needs changing.

Reply:
Yes, I wasn't too sure what to do here and tried to follow the directions as well as I could. But it looks like the pin-compatible description allows any numpy above 1.19 and below 2 for the run requirements, so other packages used that plain specification should be compatible with the 1.20+ that this package requires at runtime, right? Or are you worried about incompatibilities here during the build? The builds all seem to be passing now.

Reply:
@Tobias-Fischer think this is ok for now?

Reply:
Looks good to me, but I don't have merge rights. Sorry to ping @conda-forge/core